### ISSUE TITLE
affjax no longer in latest package sets

### ISSUE THREAD

**Issue by beebpeeb** [purescript/package-sets#507]
Is there a reason why `affjax` no longer appears in the latest package sets. Currently overcoming this with `additions` in `packages.dhall`.

**Comment by justinwoo** [comment]
https://github.com/purescript/package-sets/issues/467, I don't know when it will be updated.

**Comment by justinwoo** [comment]
There's also https://github.com/justinwoo/purescript-milkis based on whether or not you want to use window.fetch / node-fetch

**Comment by vlatkoB** [comment]
I'm trying to build `real-world-halogen` with the current set (`psc-0.13.3-20191005`) and have added to `packages.dhall` this
```dhall
let mkPackage =
      https://raw.githubusercontent.com/purescript/package-sets/psc-0.13.2-20190725/src/mkPackage.dhall sha256:0b197efa1d397ace6eb46b243ff2d73a3da5638d8d0ac8473e8e4a8fc528cf57
...
      , affjax =
          mkPackage
            [ "arrays"
            , "arraybuffer"
            , "form-urlencoded"
            , "http-methods"
            , "web-xhr"
            ]
            "https://github.com/slamdata/purescript-affjax.git"
            "v9.0.1"
```
which installs fine, but `build` of `affjax` fails. 
Reports "can't match Maybe String with String", so it seems there is a version mismatch.

@beebpeep Can you share your solution?

**Comment by beebpeeb** [comment]
@vlatkoB I solved it by using `v8.0.0` of `affjax` - because of the exact compile issue you mentioned.

**Comment by beebpeeb** [comment]
The relevant section of `packages.dhall`:

```dhall
let additions =
      { affjax =
          mkPackage
            [ "aff"
            , "argonaut-core"
            , "arraybuffer-types"
            , "foreign"
            , "form-urlencoded"
            , "http-methods"
            , "integers"
            , "math"
            , "media-types"
            , "nullable"
            , "refs"
            , "unsafe-coerce"
            , "web-xhr"
            ]
            "https://github.com/slamdata/purescript-affjax.git"
            "v8.0.0"
      }
```

**Comment by vlatkoB** [comment]
Thanks for sharing, builds and serves now. :-)

**Comment by prikhi** [comment]
There is now a [v10.0.0](https://github.com/slamdata/purescript-affjax/releases/tag/v10.0.0) release that works w/ form-urlencoded v5.

**Comment by elliotdavies** [comment]
Fixed by #534 I believe
